pkg/api/v1/client/proxy.go

```diff
@@ -96,7 +102,13 @@ func (c ProxyScriptsAPI) GetExecution(scriptID, executionID string) (execution t
 
 // ListExecutions list all executions for given script name
 func (c ProxyScriptsAPI) ListExecutions(scriptID string, limit int, tags []string) (executions testkube.ExecutionsResult, err error) {
-	uri := c.getURI("/scripts/%s/executions", scriptID)
+
+	uri := "/executions"
+
+	if scriptID != "" {
+		uri = fmt.Sprintf("/scripts/%s/executions", scriptID)
```

Comment:
remove

Reply:
@exu Do we want to change list executions command?

Reply:
good question - you have right we need it